web/src/pages/Route/transform.ts

```diff
@@ -150,6 +152,7 @@ export const transformRouteData = (data: RouteModule.Body) => {
     status,
     upstream,
     upstream_id,
+    service_id,
```

Comment:
should have a default empty string value, or maybe crashed due to undefined.length